guides/training_keras_models_on_cloud.py

```diff
@@ -1,421 +1,244 @@
-and the wide range of functionality provided within TensorFlow Cloud. We'll start
-with the simplest use-case.
-"""
+<table class="tfo-notebook-buttons" align="left">
+  <td>
+    <a target="_blank" href="https://www.tensorflow.org/cloud/tutorials/overview.ipynb"><img src="https://www.tensorflow.org/images/tf_logo_32px.png" />View on TensorFlow.org</a>
+  </td>
+  <td>
+    <a target="_blank" href="https://colab.research.google.com/github/tensorflow/cloud/blob/master/g3doc/tutorials/overview.ipynb""><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>
+  </td>
+  <td>
+    <a target="_blank" href="https://github.com/tensorflow/cloud/blob/master/g3doc/tutorials/overview.ipynb"><img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />View on GitHub</a>
+  </td>
+  <td>
+    <a href="https://storage.googleapis.com/tensorflow_docs/cloud/tutorials/overview.ipynb"><img src="https://www.tensorflow.org/images/download_logo_32px.png" />Download notebook</a>
+  </td>
+  <td>
+        <a href="https://kaggle.com/kernels/welcome?src=https://github.com/tensorflow/cloud/blob/master/g3doc/tutorials/overview.ipynb" target="blank">
+            <img width="90" src="https://www.kaggle.com/static/images/site-logo.png" alt="Kaggle logo">Run in Kaggle
+        </a>
+  </td>
+</table>
 
-"""
-## Setup
+TensorFlow Cloud is a library that makes it easier to do training and hyperparameter tuning of Keras models on Google Cloud.
 
-We'll get started by installing TensorFlow Cloud, and importing the packages we
-will need in this guide.
-"""
+Using TensorFlow Cloud's `run` API, you can send your model code directly to your Google Cloud account, and use Google Cloud compute resources without needing to login and interact with the Cloud UI (once you have set up your project in the console).
+
+This means that you can use your Google Cloud compute resources from inside directly a Python notebook: a notebook just like this one! You can also send models to Google Cloud from a plain `.py` Python script.
 
-"""shell
-pip install -q tensorflow_cloud
+## Simple example
+
+This is a simple introductory example to demonstrate how to train a model remotely using [TensorFlow Cloud](https://tensorflow.org/cloud) and Google Cloud.
+
+You can just read through it to get an idea of how this works, or you can run the notebook in Google Colab. Running the notebook requires connecting to a Google Cloud account and entering your credentials and project ID. See [Setting Up and Connecting To Your Google Cloud Account](https://github.com/tensorflow/cloud/blob/master/g3doc/tutorials/google_cloud_project_setup_instructions.ipynb) if you don't have an account yet or are not sure how to set up a project in the console.
+
+## Import required modules
 """
 
 import tensorflow as tf
-import tensorflow_cloud as tfc
+tf.version.VERSION
 
-from tensorflow import keras
-from tensorflow.keras import layers
+! pip install -q tensorflow-cloud
```

Comment:
Rather than including this line, make a note in the text paragraph above:

"""
This guide requires TensorFlow Cloud, which you can install via:
```
pip install tensorflow-cloud
```
"""

Reply:
Done.